File: app/src/main/java/demo/victormunoz/githubusers/di/module/CircleTransformationModule.java
package demo.victormunoz.githubusers.di.module;

import android.support.annotation.NonNull;

import dagger.Module;
import dagger.Provides;
import demo.victormunoz.githubusers.di.scope.ActivityScope;
import demo.victormunoz.githubusers.utils.picasso.CircleTransformation;

@SuppressWarnings("WeakerAccess")
@Module
public class CircleTransformationModule {
    @Provides
    @ActivityScope
    @NonNull
    CircleTransformation transformation(){
        return new CircleTransformation();
    }
}

File: app/src/main/java/demo/victormunoz/githubusers/di/module/CircleTransformationModule.kt
package demo.victormunoz.githubusers.di.module

import dagger.Module
import dagger.Provides
import demo.victormunoz.githubusers.di.scope.ActivityScope
import demo.victormunoz.githubusers.utils.picasso.CircleTransformation

@Module
class CircleTransformationModule {

    @Provides
    @ActivityScope
    internal fun transformation(): CircleTransformation {
        return CircleTransformation()
    }

}
